components/netlify/actions/list-site-deploys/list-site-deploys.mjs

```diff
@@ -0,0 +1,23 @@
+import netlify from "../../netlify.app.mjs";
+
+export default {
+  key: "netlify-list-site-deploys",
+  name: "List Site Deploys",
+  description: "Returns a list of all deploys for a specific site. [See docs](https://docs.netlify.com/api/get-started/#get-deploys)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    netlify,
+    siteId: {
+      propDefinition: [
+        netlify,
+        "siteId",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const response = this.netlify.listSiteDeploys(this.siteId);
```

Comment:
To optionally list more than the first page of deploys, add support for making paginated requests? I suggest using a `max` prop to limit the number of items returned (e.g., twitch [`get-clips.js`](https://github.com/PipedreamHQ/pipedream/blob/55236b3aa993cbcb545e245803d8654c6358b0a2/components/twitch/actions/get-clips/get-clips.js#L35)).

Endpoint docs: https://open-api.netlify.com/#operation/listSiteDeploys

Reply:
Addressed